tests/mock_tests/portsorch_ut.cpp

```diff
@@ -142,6 +142,14 @@ namespace portsorch_test
         };
 
         ASSERT_EQ(gPortsOrch, nullptr);
+
```

Comment:
can you please clarify how much of the changes above are covered with this test?